ui/pages/notifications/notification-components/stake/stake.tsx

```diff
@@ -92,6 +92,23 @@ const getDescription = (n: StakeNotification) => {
 export const components: NotificationComponent<StakeNotification> = {
   guardFn: isStakeNotification,
   item: ({ notification, onClick }) => {
+    const direction = DIRECTION_MAP[notification.type];
+    const amount =
+      direction === 'staked'
+        ? getAmount(
+            notification.data.stake_in.amount,
+            notification.data.stake_in.decimals,
+            {
+              shouldEllipse: true,
+            },
+          )
+        : getAmount(
+            notification.data.stake_out.amount,
+            notification.data.stake_out.decimals,
+            {
+              shouldEllipse: true,
+            },
+          );
```

Comment:
The only thing changing is stake in vs stake out.

```tsx
    const stakingProp = direction === 'staked'
      ? notification.data.stake_in
      : notification.data.stake_out;

    const amount = getAmount(
            stakingProp.amount,
            stakingProp.decimals,
            {
              shouldEllipse: true,
            },
          );
```

Reply:
So cool!